src/TTextEdit.cpp

```diff
@@ -862,6 +863,29 @@ void TTextEdit::normaliseSelection()
     }
 }
 
+void TTextEdit::expandSelectionToWords() {
+    int yind = mPA.y();
+    int xind = mPA.x();
+    for (; xind >= 0; --xind) {
+        if (mpBuffer->lineBuffer.at(yind).at(xind) == QChar::Space
+            || mpHost->mDoubleClickIgnore.contains(mpBuffer->lineBuffer[yind].at(xind))) {
+            break;
+        }
+    }
+    mPA.setX(xind+1);
+
+    yind = mPB.y();
+    xind = mPB.x();
+    for (; xind < static_cast<int>(mpBuffer->lineBuffer[yind].size()); ++xind) {
```

Comment:
Like in a previous comment, both this line and the one that is two lines down can also (better) use `mpBuffer->lineBuffer.at(yind)` instead of `mpBuffer->lineBuffer[yind]`...